gazu/files.py

```diff
@@ -108,6 +108,33 @@ def get_output_file_by_path(path):
 
 
 @cache
+def get_all_working_files_for_entity(
+        entity,
+        task=None,
+        name=None):
+    """
+    Retrieves all the working files of a given entity and specified parameters
+
+    Args:
+        entity (str / dict): The entity dict or ID.
+        task (str / dict): The task dict or ID.
+        name (str): The file name
+
+    Returns:
+        list:
+            Working files for a given entity (asset or shot), task or name
+    """
+    path = "data/entities/{entity_id}/working-files?".format(
```

Comment:
Please use `params` of `fetch_all` function instead of building a query (it's cleaner and it allows special chars).